### PR TITLE
Formatted project

### PR DESCRIPTION
**Pull request by goeppes**
https://forum.civcraft.co/t/creating-a-civcraft-java-style-guide/1610/45?u=tyrothalos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,9 @@ Stops Bukkit Minecraft server from sending enchantment, item damage and potion d
 Currently updated for Bukkit 1.7.2 but is only reliant on Bukkit API and NBT parsers so might work on other versions.
```
Comment (Maxopoly) [inline README.md]: Might want to change that Bukkit 1.7.2 to Spigot 1.10.2

File: README.md [hunk 2]
```diff
@@ -6,3 +6,9 @@ Stops Bukkit Minecraft server from sending enchantment, item damage and potion d
...
 
 Note: Enchantment spoofing is currently non-functioning.
```
Comment (Maxopoly) [inline README.md]: That works now

**Comment by Maxopoly** [comment]
You have the license both in the top directory and in resources. Also I think if we are cleaning up anyway, we might as well remove that travis.yml (what is that even for?).

**Comment by tyrothalos** [comment]
> might as well remove that .travis.yml (what is that even for?)
It's for [Travis CI](https://travis-ci.org/). I have no idea if you guys have set it up or not, but it's not particularly useful here since there aren't any tests to run. I'll remove it.

**Comment by tyrothalos** [comment]
Let me know before you merge so I can rebase this all into one commit.

**Comment by Maxopoly** [comment]
Changes seem good overall, just ping me when you have rebased it and I'll merge it.

**Comment by Maxopoly** [comment]
Also not really related to this, but because I have your attention right now:

We have a slack with all devs etc., I'll add you to that if you PM me an email adress on reddit or the forum.

**Pull request merged**